pallets/ethereum-checked/src/tests.rs

```diff
@@ -31,6 +32,14 @@ fn bounded_input(data: &'static str) -> EthereumTxInput {
         .expect("input too large")
 }
 
+// A hacky way to get around crate pub visibility of the storage item.
+#[frame_support::storage_alias]
+type Pending<T: pallet_ethereum::Config> = StorageValue<
+    pallet_ethereum::Pallet<T>,
+    Vec<(Transaction, TransactionStatus, Receipt)>,
+    ValueQuery,
+>;
+
```

Comment:
I don't feel comfortable using this hack expecially after contracts migration situation 😅. I would recommend to do it the way we used to do before, i.e make it public it our frontier fork like [this for v1.1.0](https://github.com/polkadot-evm/frontier/commit/c6a962fad3fcdfab9b0749ce1747badfa2a4493c) and old uplifts.

Also it's already public in latest upstream - https://github.com/polkadot-evm/frontier/commit/c6a962fad3fcdfab9b0749ce1747badfa2a4493c, so we will have it eventually.

Reply:
The UT is written in such a manner that it will fail if the declaration is incorrect - I tried.
But I do agree it's ugly - perhaps it's an overkill to check via UT.

I'll do as suggested though.

Also, will remove the redundant crates.